_spinnaker_install_admin_guides/install-on-aws-halyard-pod.md

```diff
@@ -0,0 +1,668 @@
+  - At least 2x worker nodes, each with at least 2 vCPUs and 4 GiB of memory. This is the bare minimum to install and run Spinnaker. If you're deploying more intermittent test workloads you will likely need more).
+
+- You have access to an S3 bucket or access to create an S3 bucket
+
+- You have access to an IAM role or user with access to the S3 bucket or can create an IAM role or user with access to the S3 bucket.
+
+This document is written with the following workflow in mind:
+
+- You have access to the EKS cluster, with `kubectl` configured and able to access the cluster, from your workstation
+- Halyard (the Spinnaker installer) will be run in the EKS cluster as a statefulset, with a single Kubernetes PersistentVolumeClaim
+
+Furthermore:
+
+- You will create AWS resources, such as service accounts, that will be permanently associated with your Spinnaker cluster
+
+# Installation Summary
+
+In order to install Spinnaker, this document covers the following things:
+
+- Creating a Kubernetes namespace to install everything in
+- Creating a Kubernetes service account with access to the namespace
+- Running Halyard (the Spinnaker installer) in the namespace
+- Creating an S3 bucket for Spinnaker to store persistent configuration in
+- Creating an IAM user that Spinnaker will use to access the S3 bucket (or alternately, granting access to the bucket via IAM roles)
+
+- Running the `hal` client interactively in the same Docker container, to:
+
+  - Build out the halconfig YAML file (`.hal/config`)
+  - Configure Spinnaker with the IAM credentials and bucket information
+  - Turn on other recommended settings (artifacts and http artifact provider)
+  - Install Spinnaker
+  - Expose Spinnaker
+
+# Connect to the Kubernetes cluster
+
+Spinnaker needs a credential to talk to Kubernetes, so you must create a service account in your Kubernetes cluster.
+
+## Connecting to an EKS cluster
+
+If you're using an EKS cluster, you must be able to connect to the EKS cluster. This assumes you have already configured the `aws` CLI with credentials and a default region / availability zone (see installation directions [here](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html) and configuration directions [here](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html))
+
+1. Create the local working directory on your workstation. For the purposes of this document, we will be using `~/aws-spinnaker`, but this can be any persistent directory on any Linux or OSX machine.
+
+	```bash
+	mkdir ~/aws-spinnaker
+	cd ~/aws-spinnaker
+	```
+
+2. If you have access to the role that created the EKS cluster, you can create a kubeconfig with access to your Kubernetes cluster with this command:
```

Comment:
Do we need to link , or provide an alternative for those that do not have access to the role?  
maybe something as simple as contact your admin for a kubeconfig?  or alternatively, link to AWS for creating a new cluster?

Reply:
oh i just see line 102... so nm.